src/wdio/browser.ts

```diff
@@ -127,4 +127,17 @@ export class Browser {
         const caps = browser.capabilities as any;
         return caps.platformName;
     }
+
```

Comment:
I miss updating the package.json file to move to next version, so a new lib version can be published upon merge.

Reply:
Updated accordingly